README.md

```diff
@@ -22,20 +22,55 @@ Then you can install `fpm`:
 gem install --no-ri --no-rdoc fpm
 ```
 
-### install yarn for build dashboard
+### run build tool script:
+The version of APISIX is hard-code in `run.sh` now, you can change it by yourself.
+
+```
+./run.sh
+```
+
+The RPM package will be in `/tmp/rpm`.
```

Comment:
I think we can delete this description. 

show it in the output of script,  This should be enough.

Reply:
fixed.